apis/supervisor/config/v1alpha1/types_federationdomain.go.tmpl

```diff
@@ -58,6 +207,32 @@ type FederationDomainSpec struct {
 	// TLS configures how this FederationDomain is served over Transport Layer Security (TLS).
 	// +optional
 	TLS *FederationDomainTLSSpec `json:"tls,omitempty"`
+
+	// IdentityProviders is the list of identity providers available for use by this FederationDomain.
+	//
+	// An identity provider (e.g. OIDCIdentityProvider or LDAPIdentityProvider) describes how to connect to a server,
```

Comment:
I'm wondering if all of the detail here is in the best place for clarity.  In summary:

- An IdentityProvider ... describes how to connect to a server...
- In contrast, IdentityProviders describes how to use that normalized identity in a group of Kubernetes clusters...